src/functions-and-arrays.js

```diff
@@ -1,18 +1,109 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(number1, number2){
+  return Math.max(number1, number2);
+}
+
+
+
 // Iteration #2: Find longest word
+
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+function findLongestWord(words) {
+  let word ='';
+  if (!words.length){
+    return null;
+  } else for (let i = 0; i < words.length; i++) {
```

Comment:
I would wrap the for loop by the curly braces (not used right now) of the else clause.